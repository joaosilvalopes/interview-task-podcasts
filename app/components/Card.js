import styled from 'styled-components';

const Card = styled.div`
    box-shadow: 0rem 0.16rem 0.32rem gray;
`;

export default Card;
