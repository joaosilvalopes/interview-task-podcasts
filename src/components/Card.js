import styled from 'styled-components';

const Card = styled.div`
    box-shadow: 0rem 0.1rem 0.2rem gray;
`;

export default Card;
