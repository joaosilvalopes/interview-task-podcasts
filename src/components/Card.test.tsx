import { render } from '@testing-library/react';

import Card from './Card';

it('should render correctly', async () => {
    const { asFragment } = render(<Card />);
  
    expect(asFragment()).toMatchSnapshot();
});
