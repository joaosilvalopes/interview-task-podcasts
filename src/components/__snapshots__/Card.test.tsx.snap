// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bcXHqe bFLobp"
  />
</DocumentFragment>
`;
